.gitignore

```diff
@@ -40,3 +40,4 @@ launchSettings.json
 msbuild.ProjectImports.zip
 StyleCop.Cache
 UpgradeLog.htm
+build-sentinel
```

Comment:
Recommend _not_ doing this. Place the file in `$(BaseIntermediateOutputPath)` instead